src/k8s-runtime/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.0.0
+++++++
+* Use stable ARM API version `2024-08-01` *
```

Comment:
It seems like there are some breaking changes on some command signatures, right? Such as ` If so, please mark it in the history notes.
In addition, may I ask have you evaluate the impact of these breaking changes? We may need to consider releasing it in the breaking change window to reduce the impact on customers

Reply:
![Uploading image.png…]()